plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -90,8 +89,9 @@ class Tracker:
         being in either cartesian, cylindrical, or spherical coordinates
         based on its units. Valid geometries are:
 
+        * Cartesian (x,y,z) : (meters, meters, meters)
         * Cartesiaical (r, theta, z) : (meters, radians, meters)
```

Comment:
```suggestion
        * Cylindrical (r, theta, z) : (meters, radians, meters)
```
Is this supposed to be "cylindrical" here?

Reply:
>_>